community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/util.py

```diff
@@ -459,6 +459,8 @@ def install_custom_scripts(check_hash=False):
         tokens = [
             "prolog",
             "epilog",
+            "task_prolog",
```

Comment:
nit. Instead of copy pasting we could do something like this:

```py
all_prolog_tokens = ["prolog", "epilog", "task_prolog", "task_epilog"]
...
tokens = ["controller"] + all_prolog_tokens
....
tokens = [f"nodeset-{lookup().node_nodeset_name()}"] + all_prolog_tokens
```

Reply:
Thank you. Fixed.